tasks/zadanie_14.php

```diff
@@ -43,7 +43,34 @@
 
 $products = $params[0]; // tej linijki nie ruszamy :)
 
+$lp = 0;
 
-echo "Lp. 1 | Laptop | 1 szt. | 3000 zł | VAT 23% | brutto: 3000 zł\n";
-echo "Lp. 2 | Monitor | 2 szt. | 700 zł | VAT 23% | brutto: 1400 zł\n";
-echo "\nSUMA BRUTTO: 4400 zł";
\ No newline at end of file
+
+foreach ($products as $key => $value){
+	
+    if (is_array($value)) {
+    	 
+    	 	$name = "";
+        	$qty = 0;
+        	$vat = 0;
+        	$price = 0;
+        	
+        	
+        foreach ($value as $key2 => $value2 ){
```

Comment:
Jak wyżej zdefiniujesz to, to potem zamiast tego całego foreach wystarczy samo:
```suggestion
       $amount	= $qty * $price ;
       $sum = $sum + $amount;
```
:wink: